api/src/main/java/org/openmrs/obs/handler/AbstractHandler.java

```diff
@@ -59,46 +55,31 @@ public AbstractHandler() {
 	 * @return File that the complex data should be written to
 	 */
 	public File getOutputFileToWrite(Obs obs) throws IOException {
-		// Get the title and remove the extension.
+		// Get the title and the extension
```

Comment:
please remove comments, your code is nicely readable ;)

Reply:
I mean all related to this ticket. I mean these comments are just duplicating the code, therefore they should go. I personally only very rarely add comments.

Reply:
I can see why, although I like comments when reading and writing code because they provide some order, or "story", like headlines (even though headlines only condense the content of the following text as well). I also find code blocks I'm looking for faster by using the comments, even if it's not complicated code. I suppose that's a matter of personal preference. I'll delete them, of course.